src/components/SkillPage/SkillListing.js

```diff
@@ -187,6 +193,31 @@ class SkillListing extends Component {
         this.setState({ showAuthorSkills: false });
     };
 
+    parseDate = dtstr => {
+        // replace anything but numbers by spaces
+        dtstr = dtstr.replace(/\D/g,' ');
+        // trim any hanging white space
+        dtstr = dtstr.replace(/\s+$/,'');
+        // split on space
+        var dtcomps = dtstr.split(' ');
+        // not all ISO 8601 dates can convert, as is
+        // unless month and date specified, invalid
+        if (dtcomps.length < 3) {
+            return 'invalid date';
```

Comment:
It would be better if the `I` is capitalised. `Invalid date`

Reply:
Right, changing. :+1: